hera_pspec/pspecbeam.py

```diff
@@ -751,14 +751,14 @@ def power_beam_sq_int(self, pol='pI'):
         # type check
         if isinstance(pol, str):
             pol = uvutils.polstr2num(pol, x_orientation=self.x_orientation)
-            
+
         if pol in self.OmegaPP.keys():
             return self.OmegaPP[pol]
         else:
-            raise KeyError("OmegaPP not specified for polarization '%s'. " 
+            raise KeyError("OmegaPP not specified for polarization '%s'. "
                            "Available polarizations are: %s" \
                            % (pol, self.available_pols))
-    
+
```

Comment:
I can't tell--did you actually change any code in pspecbeam.py?

Reply:
I didn't really change any code.